lib/axiosHttpClient.ts

```diff
@@ -16,6 +16,18 @@ import * as http from "http";
 import * as https from "https";
 import { URLBuilder } from "./url";
 
+const client = axios.create();
+
+// This hack is still required with 0.19.0 version of axios since axios tries to merge the
+// Content-Type header from it's config["<method name>"] where the method name is lower-case,
+// into the request header. It could be possible that the Content-Type header is not present
+// in the original request and this would create problems while creating the signature for
+// storage data plane sdks.
+client.interceptors.request.use((config: AxiosRequestConfig) => ({
+  ...config,
+  method: (config.method as Method) && (config.method as Method).toUpperCase() as Method
+}));
```

Comment:
This change LGTM.

Reply:
@amarzavery  Can we have this added to latest 1.x.x release so that the memory leak issue can be resolved in `@azure` packages which are still relying on versions `^1.8.1` of `@azure/ms-rest-js`

Reply:
Yes. If everything goes well, then the next version in the 1.x release train would be 1.8.14. @daviwil is the current owner and should be able to provide more details on this.

Reply:
Thanks @amarzavery